lib/src/lifecycle_module.dart

```diff
@@ -228,6 +237,9 @@ abstract class LifecycleModule extends SimpleModule
       _didLoadController.add(this);
       _transitionFuture = null;
       completer.complete();
+    }).catchError((Error error) {
```

Comment:
I think this should be untyped or `Object` - this could also be an `Exception` and those two don't share any inheritance